src/observation/subscribeToQueryWithColumns/index.js

```diff
@@ -60,7 +60,7 @@ export default function subscribeToQueryWithColumns<Record: Model>(
   // on the other, it would be good to have source provided as Observable, not Query
   // so that we can reuse cached responses -- but they don't have compatible format
   const [subscribeToSource, asyncSource] = query.hasJoins
-    ? [observer => subscribeToQueryReloading(query, observer, true), true]
+    ? [observer => subscribeToQueryReloading(query, observer, true), false]
```

Comment:
this is now slightly broken (performance/flicker optimization from v0.15 is reverted if you DON'T use synchronous mode). Not sure if this should be a blocker. Your call.

Reply:
Hmm I think this is ok. Updating library is not mandatory so if someone feels strongly about this they can use v0.15 until this issue is resolved.